### ISSUE TITLE
Skip Show Grants

### ISSUE THREAD

**Issue by nialloriordan** [Titan-Systems/titan#122]
I'm testing Titan at work to centralise our DDL processes however I'm running into a few issues.

Calling SHOW TAGS or SHOW GRANTS for all our roles takes an exceedingly long time to return and isn't necessary for the resources I'm trying to create - tables and grants.

To provide more context we have over 100 roles and `SHOW GRANTS TO ROLE SYSADMIN` returns `(56515 rows, 322.80s)`.

This process eventually failed with:
```
[<my-email>:SYSADMIN] > SHOW GRANTS TO ROLE <my-role>   (0 rows, 0.10s)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
....
    raise OrphanResourceException(
titan.exceptions.OrphanResourceException: Blueprint is missing a database but includes resources that require a database or schema
```

I'm calling titan via the cli `python -m titan plan --config <path-to-yaml-file>`

Is there a way to skip this step or specify which roles/tags to check? Alternatively it won't be possible to use this tool.

**Comment by teej** [comment]
`SHOW TAGS` is used to determine if your account is running on Standard Edition or not. I'll go find an alternative.

`SHOW GRANTS TO SYSADMIN` is used for execution planning, but could probably be skipped, I'll take a look.

That `OrphanResourceException` for databases can fire earlier, I'll investigate.

I'll try to get these fixed in a release this week.

**Comment by littleK0i** [comment]
You may get edition by running `SELECT SYSTEM$BOOTSTRAP_DATA_REQUEST('ACCOUNT')` and parsing JSON.

Edition is in `accountInfo.serviceLevelName`.

**Comment by teej** [comment]
Fixed in [v0.10](https://github.com/Titan-Systems/titan/releases/tag/v0.10.0)
